source/threads.tex

```diff
@@ -4838,7 +4838,7 @@
 \tcode{\placeholdernc{DECAY_COPY}(std::forward<Args>(args))...)}
 (\ref{func.require}, \ref{thread.thread.constr})
 as if in a new thread of execution represented by a \tcode{thread} object
-with the calls to \tcode{\placeholdernc{DECAY_COPY}()} being evaluated in the thread that called \tcode{async}.
+with the calls to \tcode{\placeholdernc{DECAY_COPY}} being evaluated in the thread that called \tcode{async}.
```

Comment:
@jwakely: Then get rid of the "nc", too. That was only needed for the spacing relative to the parens.